sdk/python/tests/integration/registration/test_registry.py

```diff
@@ -247,7 +247,8 @@ def test_modify_feature_views_success(test_registry):
     )
 
     request_source = RequestSource(
-        name="request_source", schema={"my_input_1": ValueType.INT32}
+        name="request_source",
+        schema=[Field(name="my_input_1", dtype=PrimitiveFeastType.INT32)],
```

Comment:
Can you also have a test for the old to make sure it still works?

Reply:
Done.